src/management/endpoint.c

```diff
@@ -85,6 +85,10 @@ static int endpoint_create_process_data(struct controller_api *endpoint_create)
 	struct json_object *json_obj = NULL;
 	struct json_object *json_fs_name_obj = NULL;
 	struct json_object *json_endpoint_options_obj = NULL;
+	struct json_object *json_resp_obj = NULL;
+        uint16_t fs_id = 0;
```

Comment:
please take care of the indentation here

Reply:
It looks correct on Linux. May be github issue.